packages/kbn-flot-charts/lib/jquery_flot_axislabels.js

```diff
@@ -380,7 +380,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
                         if (!opts.axisLabelUseHtml &&
                             navigator.appName == 'Microsoft Internet Explorer') {
                             var ua = navigator.userAgent;
-                            var re  = new RegExp("MSIE ([0-9]{1,}[\.0-9]{0,})");
+                            var re  = new RegExp("MSIE ([0-9]{1,}[\\.0-9]{0,})");
```

Comment:
Not sure why Ops owns flot-charts, but I think we should actually remove the `\` instead of escaping it

Reply:
As the code is currently the period isn't escaped and so the character class matches all characters. I assume you want the character class to only match a literal period (`.`) along with the numbers 0 to 9 right? If so it needs to be escaped with a double backslash (`\\`). Or maybe I have misunderstood its purpose?

Reply:
What make it hard to read is because it's using `RegExp` with a string, instead of the regex literal syntax, where only a single backslash is needed:

```suggestion
                            var re  = /MSIE ([0-9]{1,}[\.0-9]{0,})/;
```

Reply:
Oh wait, you're right 🤦 

Inside of character class periods are treated as their literal value. So it can just be removed yes

Reply:
I've pushed a new commit with a fix 👍